### PR TITLE
posix-manpages: new port

### PR DESCRIPTION
**Pull request by mohd-akram**
###### Tested on
macOS 10.14.3 18D109
Xcode 10.1 10B61

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?

### CODE REVIEW

File: devel/posix-manpages/Portfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                posix-manpages
+version             2013
+categories          devel
+platforms           darwin
+supported_archs     noarch
+license             Other
```
Comment (mf2k) [inline devel/posix-manpages/Portfile]: That's not a helpful value for the `license`. It won't be distributable. See https://trac.macports.org/wiki/PortfileRecipes#license for assistance.

File: devel/posix-manpages/Portfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+checksums           rmd160  b19d97fad34a199ebb34bb3b4085e1439f63de77 \
+                    sha256  77de3cc94de6312d2b18d4bfd53b8922456688212dbfdee3ede6191f400a4b29 \
+                    size    1489221
+
+use_configure       no
+build {}
+destroot {
+    xinstall -d ${destroot}${prefix}/share/man/man0
+    system "cd ${worksrcpath}/man0p; ls | xargs -I {} install {} \
```
Comment (mf2k) [inline devel/posix-manpages/Portfile]: Use `system -W ${worksrcpath}/man0p ...` instead.

File: devel/posix-manpages/Portfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        ${destroot}${prefix}/share/man/man0/posix:{}"
+    xinstall -d ${destroot}${prefix}/share/man/man1
+    system "cd ${worksrcpath}/man1p; ls | xargs -I {} install {} \
```
Comment (mf2k) [inline devel/posix-manpages/Portfile]: As above, use `system -W ...` instead.

File: devel/posix-manpages/Portfile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        ${destroot}${prefix}/share/man/man1/posix:{}"
+    xinstall -d ${destroot}${prefix}/share/man/man3
+    system "cd ${worksrcpath}/man3p; ls | xargs -I {} install {} \
```
Comment (mf2k) [inline devel/posix-manpages/Portfile]: As above, use `system -W ...` instead.

File: devel/posix-manpages/Portfile (hunk 1 above)
Comment (mohd-akram) [reply devel/posix-manpages/Portfile]: I've changed it to `Permissive` based on the discussion [here](https://lwn.net/Articles/581858/).

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**